pkgs/tools/virtualization/ec2instanceconnectcli/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, boto3 }:
+
+buildPythonPackage rec {
+  pname = "ec2instanceconnectcli";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-VaCyCnEhSx1I3bNo57p0IXf92+tO1tT7KSUXzO1IyIU=";
+  };
+
+  propagatedBuildInputs = [ boto3 ];
```

```suggestion
  propagatedBuildInputs = [ boto3 ];
  
  # has no tests
  doCheck = false;
  
  pythonImportsCheck = [ "ec2instanceconnectcli" ];
```

I am not sure on the imports check. If this is not importable it can be skipped.

---

Looks like it is, thanks.